### DISCUSSION TITLE
[2021-09-16] 아픈 동물 찾기

### DISCUSSION THREAD

**Discussion by hyk339** [pyunji/Hyundai-ITnE-Team_QuadCore#134] [SQL]
[아픈 동물 찾기](https://programmers.co.kr/learn/courses/30/lessons/59036)

**Comment by chaeyeon4u** [comment]
[아픈 동물 찾기 solved](https://github.com/pyunji/Hyundai-ITnE-Team_QuadCore/blob/master/ycy/homework/SQL/PR_59036.sql)

**Comment by hyk339** [comment]
[아픈 동물 찾기 solved](https://github.com/hyk339/Hyundai-ITnE-Team_QuadCore/blob/master/khy/sql/PR_59036.sql)

**Comment by leechang4** [comment]
[아픈 동물 찾기 solved](https://github.com/leechang4/Hyundai-ITnE-Team_QuadCore/blob/master/lch/SQL/PR%2059036.sql)
